sdk/batch/microsoft-azure-batch/src/test/java/com/microsoft/azure/batch/BatchIntegrationTestBase.java

```diff
@@ -87,6 +87,8 @@ private static void initTestMode() throws IOException {
                 testMode = TestBase.TestMode.RECORD;
             } else if (azureTestMode.equalsIgnoreCase("Playback")) {
                 testMode = TestBase.TestMode.PLAYBACK;
+            } else if (azureTestMode.equalsIgnoreCase("Live")) {
+                testMode = TestMode.RECORD;
```

Comment:
not `TestMode.LIVE`?

Reply:
This is another enumeration of the same name TestMode but it only supports RECORD and PLAYBACK.